tests/cases/standalone/common/timestamp/timestamp.sql

```diff
@@ -1,3 +1,5 @@
+SELECT timestamp '    2017-07-23    13:10:11   ';
+
```

Comment:
I've updated [dummy.sql](https://github.com/GreptimeTeam/greptimedb/blob/d7002caca75f7123d155dbb0eb621e37b46b887a/tests/cases/standalone/common/select/dummy.sql#L1) / [dummy.result](https://github.com/GreptimeTeam/greptimedb/blob/d7002caca75f7123d155dbb0eb621e37b46b887a/tests/cases/standalone/common/select/dummy.result#L1) and have the following result 

```
[dummy.sql]
select TO_UNIXTIME('    2023-03-01T06:35:02Z    ');
...
[dummy.result]
select TO_UNIXTIME('    2023-03-01T06:35:02Z    ');

+---------------------------------------------------+
| to_unixtime(Utf8("    2023-03-01T06:35:02Z    ")) |
+---------------------------------------------------+
| 1677652502                                        |
+---------------------------------------------------+
``` 
Not sure if that's a valid usecase though. Let me know if you'd like this to be added

Reply:
This looks fine to me 👍